registry/data-models/models.py

```diff
@@ -0,0 +1,168 @@
+      |      1:n  +------------------+           +------------------+
+      |                 ｜                             ｜          |
+      |                 ｜ 1:n                         ｜1:n       |
+      |                 +------------------------------+          |
+      |                               ｜                           |
+      |                               ｜1:n                       \|/ has
+      |   +--------------+    +---------------+                 +----------------+
+      |   |Transformation|----|    Feature    | --------------- |   DataSource   |            
+      |   +--------------+ has+---------------+                 +----------------+
+      |                              /|\   |                     /|\           |extends
+      |                               |    |                      |            |
+      |                               |    +----------------------|---------+  |
+      |                               |                           |         |  |
+      |          +--------------------------------------+         |      has| \|/
+      |          |                                      |         |        +----------+
+      |          |extends                               |extends  |has     |  Source  |
+      |    +----------------+                      +-----------------+     +----------+
+      |    | DerivedFeature |                      |  AnchorFeature  |          /|\
+      |    +----------------+                      +-----------------+           |
+      |            |                                                             |
+      |            |                                                             |extends
+      |            |                                                       +------------------+
+      |            +-------------------------------------------------------|   FeatureSource  |
+      |                                                                has +------------------+
+      |                                                                             |
+      +-----------------------------------------------------------------------------|
+                      
+                      
+                      
+                      
+                      
+                      
+"""
+
+
+class FeatureId(BaseModel):
+    """
+    Id for Feature, it's unique ID represents Feature.
+    """
+    pass
+
+
+class FeatureNameId(BaseModel):
+    """
+    Id for FeatureName, it's unique ID represents FeatureName.
+    """
+    pass
+
+
+class AnchorId(BaseModel):
+    """
+    Id for Anchor, it's unique ID represents Anchor.
+    """
+    pass
+
+
+class ProjectId(BaseModel):
+    """
+    Id for Project, it's unique ID represents Project.
+    """
+    pass
+
+
+class Source(BaseModel):
+    """
+    Type of FeatureValue.
+    It defines where the feature is extracted from.
+    """
+    pass
+
+
+class DataSource(Source):
```

Comment:
pydoc for each data model please, including the `FeatureSource` below

Reply:
Added doc for all data models